TheBTeam.BLL/Services/TransactionService.cs

```diff
@@ -111,12 +53,26 @@ public static void ApplyTransaction(Transaction transaction, User user)
         }
         public List<Transaction> SearchTransactionByUser(string id)
         {
-            return Transaction.Where(t => t.User.Id == id).ToList();
+            return _transactions.Where(t => t.User.Id == id).ToList();
         }
         public static List<Transaction> SearchTransactionByType(TypeOfTransaction type, List<Transaction> transactions)
         {
             var transactionsByType = transactions.Where(t => t.Type == type).ToList();
             return transactionsByType;
         }
+
+        public Transaction GetTransactionByUser(string id)
```

Comment:
so here is GetTransactionByUser and later on is just Delete, we should keep one pattern
According to the fact, that we are working here on non-static class I thing less is better like GetByUser